api.planx.uk/modules/analytics/metabase/shared/types.ts

```diff
@@ -0,0 +1,40 @@
+export interface Input {
+  teamName: string;
+  originalDashboardId: number;
+  /** The name of the filter to be updated */
+  filter: string;
+  /** The value of the new filter; updateFilter() only supports strings right now */
+  filterValue: string;
+}
+
+export function validateInput(input: unknown): input is Input {
```

Comment:
This looks like it's currently unused - is that right?

In terms of validation, it would be best to follow the existing pattern in the API.

 → Incoming request of type `unknown` → 
 → Passes through existing `validate()` middleware in route → 
 → Using Zod, is checked against a provided schema → 
 → Validated payload is automatically passed along as `res.locals.parsedReq` → 
 → Controller and Service have both type safety and a validated payload